src/frontend/screens/CartScreen/CartScreen.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+function CartScreen() {
+  return (
+    <div
+      className="main-content"
+      style={{ color: "white", marginTop: "10rem", textAlign: "center" }}
```

Comment:
Use class name instead of inline styling

Reply:
You're right. This is just the placeholder styling. Will be anyways removed later.